packages/sql/src/migration/migration-provider.ts

```diff
@@ -51,24 +51,62 @@ export class MigrationProvider {
         return migrationsPerDatabase;
     }
 
+    private async registerTsx() {
+        const esm = isEsm();
+        if (esm) {
+            // @ts-ignore
+            const { register } = await import('tsx/esm/api');
+            register();
+        } else {
+            const { register } = require('tsx/cjs/api');
+            register();
+        }
+    }
+
+    async addDatabase(path: string): Promise<void> {
+        await this.registerTsx();
+
+        const exports = Object.values((await import(join(process.cwd(), path)) || {}));
+        if (!exports.length) {
+            throw new Error(`No database found in path ${path}`);
+        }
+
+        let databaseInstance: Database | undefined;
+        let foundDatabaseClass: ClassType<Database> | undefined;
+
+        for (const value of exports) {
+            if (value instanceof Database) {
+                databaseInstance = value;
+                break;
+            }
+            if (Object.getPrototypeOf(value) instanceof Database) {
```

Comment:
I'm having issues with nominal checks during local development.
I'm not entire sure if it's because of tsx or npm-local-development.
```suggestion
            if (value.name === Database.name) {
```